### PR TITLE
Link updates and tidy up

### PR DESCRIPTION
**Pull request by franmoore05**
### What

Tidy up and general update of platform training guide.

### How to review

Ensure the changes make sense.

### Who can review

Anyone.

### CODE REVIEW

File: training/platform-services/DEPLOYMENT_AND_RUNNING.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+Furthermore, Concourse helps us to identify if an application has been fully deployed with any new changes implemented and can provide information on the version of the application which is currently running in the `develop` and `production` environments.
+
+Please note that when deploying changes to the `production` environment (the live website), you need to **manually** deploy the application by doing the following:
+
+1. Go to [Concourse][concourse-ons]
+2. Navigate to the application
+3. Click on `production-ship-it`
+4. <div>Click on the <img src="images/concourse-build.png" title="Concourse Build" width=4% height=4%></div>
+5. Login to concourse as `dp` user (if you do not have the login, please ask anyone in the team)
```
Comment (redhug1) [inline training/platform-services/DEPLOYMENT_AND_RUNNING.md:21]: Should this step come before some of the above ?

**Review by dionisislolas — DISMISSED** [review]

**Review by redhug1 — COMMENTED** [review]
just the one observation

File: training/platform-services/DEPLOYMENT_AND_RUNNING.md (hunk 1 above)
Comment (franmoore05) [reply training/platform-services/DEPLOYMENT_AND_RUNNING.md]: This has now been amended

**Review by redhug1 — APPROVED** [review]
LGTM

**Pull request merged**